EpsilonAuxBmsV2/Tests/tests/SendAuxTripTaskTest/main.c

```diff
@@ -0,0 +1,23 @@
+//Include all files required for your functions to run
+#include <stdio.h>
+
+//Include required .h file for unity
+#include "unity.h"
+#include "stm32f4xx_hal.h"
+#include "SendAuxTripTaskTest.h"
+
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+int main (void)
+{
+
+    runSendAuxTripTaskTest();
```

Comment:
Need to return what this returns as `main` returns an `int` so should be: `return runSendAuxTripTaskTest()`